### PR TITLE
Allows opening resource manager folders with double-click

### LINKED ISSUE #5276: Cannot open folders in new Resource Manager module via right-hand panel
## Description of bug
The new Resource Manager module shows files and folders in the right-hand panel. However folders cannot be opened via the right-hand panel. Clicking a folder (anywhere in the row) only marks it as selected with the radio button control in the left column. After that, there seems to be no way to open the folder (nothing appear in the toolbar above, also nothing in the right-click menu).

## Steps to reproduce
List the precise steps to reproduce the bug:
1. Go to the new Resource Manager module
2. Click a folder in the right-hand panel (anywhere in the row).
3. The folder is marked selected, not opened and no option appears to open the folder

## Expected behavior
There should be an intuitive way to open folders via the right-hand panel (as well as selecting them for editing). Only being able to navigate folders via the left-hand panel is restrictive. In the old Digital Asset Management (Telerik) module files and folders could be selected for edit operation by checking the checkbox, and opened by clicking the file/folder name - this was very intuitive.

## Screenshots
New Resource Manager module:
image
Old Digital Asset Management (Telerik) module
image

## Affected version
* [ X ] 09.11.00 release candidate 4

### PR DESCRIPTION
**Pull request by valadas**
Closes #5276

### CODE REVIEW

File: DNN Platform/Modules/ResourceManager/ResourceManager.Web/src/components/dnn-rm-folder-list-item/dnn-rm-folder-list-item.tsx [hunk 1]
```diff
@@ -32,6 +32,16 @@ export class DnnRmFolderListItem {
     }
   }
 
+  @Listen("dnnRmFolderDoubleClicked", {target: "document"})
+  handleFolderDoubleClicked(e: CustomEvent<number>) {
+    if (e.detail == Number.parseInt(this.folder.data.key)) {
+      this.dnnRmFolderListItemClicked.emit(this.folder);
+      this.handleUserExpanded();
+      this.expanded = true;
+      console.log(e);
```
Comment (david-poindexter) [inline DNN Platform/Modules/ResourceManager/ResourceManager.Web/src/components/dnn-rm-folder-list-item/dnn-rm-folder-list-item.tsx:41]:
```suggestion
```

**Review by david-poindexter — CHANGES_REQUESTED** [review]
Looks great - thanks @valadas - just one minor clean up item.

File: DNN Platform/Modules/ResourceManager/ResourceManager.Web/src/components/dnn-rm-folder-list-item/dnn-rm-folder-list-item.tsx (hunk 1 above)
Comment (valadas) [reply DNN Platform/Modules/ResourceManager/ResourceManager.Web/src/components/dnn-rm-folder-list-item/dnn-rm-folder-list-item.tsx]: Good catch

**Review by david-poindexter — APPROVED** [review]

**Review by bdukes — APPROVED** [review]
LGTM

**Pull request merged**